### PR TITLE
fix(design): fix bug where daff-backdrop had the wrong z-index on IE

### PR DESCRIPTION
**Pull request by lderrickable**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

Fixed: https://github.com/graycoreio/daffodil/issues/118

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by lderrickable** [comment]
> @lderrickable How do you set a default backdrop instead of the transparent variant? / why is the transparent variant needed?

default backdrop is
```
<daff-backdrop (backdropClicked)="backdropClicked()"></daff-backdrop>
```

Transparent variant is needed in case you want the backdrop to be transparent.

File: libs/design/src/molecules/sidebar/sidebar-viewport/sidebar-viewport.component.html [hunk 1]
```diff
@@ -5,9 +5,8 @@
 
   <daff-backdrop
     class="daff-sidebar-viewport__backdrop"
-    *ngIf="hasBackdrop"
-    [show]="opened"
-    [backdropIsVisible]="backdropIsVisible"
+    *ngIf="hasBackdrop && _opened"
+    [transparent]="backdropIsVisible"
```
Comment (xelaint) [inline libs/design/src/molecules/sidebar/sidebar-viewport/sidebar-viewport.component.html]: should be `!backdropIsVisible`

File: libs/design/src/molecules/modal/modal/modal.component.html [hunk 2]
```diff
@@ -3,9 +3,7 @@
     <ng-content></ng-content>
   </div>
 
-  <daff-backdrop
-    class="daff-modal__backdrop"
-    [show]="show"
-    [backdropIsVisible]="backdropIsVisible"
+  <daff-backdrop class="daff-modal__backdrop" *ngIf="show"
+    [transparent]="backdropIsVisible"
```
Comment (xelaint) [inline libs/design/src/molecules/modal/modal/modal.component.html]: should be `!backdropIsVisible`

Comment (lderrickable) [reply libs/design/src/molecules/modal/modal/modal.component.html]: fixed and squashed

**Pull request merged**